<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>881</width>
     <height>731</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
      <item>
       <widget class="QGroupBox" name="image_viewer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Image Viewer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="index_navigator">
        <property name="title">
         <string>Index Navigator</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>591</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QScrollBar" name="idx_scroll">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>30</number>
            </property>
            <item row="1" column="0">
             <widget class="QLineEdit" name="start_idx_ledit"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>end</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>step</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>start</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="end_idx_ledit"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="step_size_ledit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,3,1,2">
      <item>
       <widget class="QGroupBox" name="log_gbox">
        <property name="title">
         <string>Log</string>
        </property>
        <widget class="QTextBrowser" name="log_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>241</width>
           <height>51</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="curr_dir_gbox">
        <property name="title">
         <string>Current Directory</string>
        </property>
        <widget class="QLabel" name="curr_dir_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>241</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Current Index</string>
        </property>
        <widget class="QLabel" name="curr_idx_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="curr_files_gbox">
        <property name="title">
         <string>Currently Loaded Files</string>
        </property>
        <widget class="QTableWidget" name="curr_files_table">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>241</width>
           <height>201</height>
          </rect>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="viewer_options">
        <property name="title">
         <string>Viewer Options</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>241</width>
           <height>57</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="flip_horizontally_cbox">
            <property name="text">
             <string>Flip Horizontally</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="flip_vertically_cbox">
            <property name="text">
             <string>Flip Vertically</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="crop_etc_cbox">
            <property name="text">
             <string>Crop / Adjust Vmin, Vmax</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="clear_data_btn">
          <property name="text">
           <string>Clear Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_loading_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cancel Loading</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_file_btn">
          <property name="text">
           <string>Load File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_dir_btn">
          <property name="text">
           <string>Load Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_btn">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>idx_scroll</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>Form</receiver>
   <slot>scrolled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>750</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flip_horizontally_cbox</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>show_img()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>822</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>925</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flip_vertically_cbox</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>show_img()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>913</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_idx_ledit</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>adjust_idx_range()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>724</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>step_size_ledit</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>adjust_idx_range()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>724</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>778</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>end_idx_ledit</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>adjust_idx_range()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>766</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scrolled()</slot>
  <slot>show_img()</slot>
  <slot>adjust_idx_range()</slot>
 </slots>
</ui>
